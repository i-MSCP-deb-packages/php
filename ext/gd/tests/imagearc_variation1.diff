001+ edb768b4c16d65e6d4418b2020b97dea
001- ed2c8427a9922dfd8a105f10a88a0d20