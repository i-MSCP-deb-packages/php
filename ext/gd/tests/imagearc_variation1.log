
---- EXPECTED OUTPUT
ed2c8427a9922dfd8a105f10a88a0d20
---- ACTUAL OUTPUT
edb768b4c16d65e6d4418b2020b97dea
---- FAILED
