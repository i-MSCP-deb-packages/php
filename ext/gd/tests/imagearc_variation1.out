edb768b4c16d65e6d4418b2020b97dea